### DISCUSSION TITLE
Consider renaming the project

### DISCUSSION THREAD

**Discussion by fwbrasil** [microsoft/jfr-streaming#12] [General]
I was expecting that this library could bring the JFR streaming features to older JDKs but its scope seems more limited than that and it provides another way to manage JFR executions with a similar feature set to jcmd.

Renaming the library to avoid confusion with the JFR streaming features in more recent JDKs might be beneficial to the JVM community.

**Comment by dsgrieve** [comment]
Yeah. The name came out of using the FlightRecorderMXBean stream API. Really this is a rough equivalent of jdk.jfr.Recording, which you don't have in jdk 1.8. That and you can use this library to control JFR on a local or remote JVM. 

Do you have any suggestions for a better name?
